### PR TITLE
add ithaca to retired systems list

### PR DESCRIPTION
**Pull request by abbycross**

### CODE REVIEW

**Comment by abbycross** [comment]
Waiting on the final retirement date, then ready for review/merge

**Comment by Eric-Arellano** [comment]
Is this still relevant?

**Comment by abbycross** [comment]
Still waiting to hear official date of retirement. Will ping relevant parties again.

**Comment by abbycross** [comment]
https://github.ibm.com/IBM-Q-General/client-comms/issues/423

File: docs/run/retired-systems.mdx [hunk 1]
```diff
@@ -15,6 +15,7 @@ The following systems have been retired. For the full list of available systems,
 
 | System name       | Qubit count | Retirement date (Year - month - day) |
 | ----------------- | ----------- | --------------- |
+| ibm_ithaca        | **65**      | 2023-11-28      |
```
Comment (abbycross) [inline docs/run/retired-systems.mdx:18]:
```suggestion
| ibm_ithaca        | **65**      | 2024-01-25      |
```

File: docs/run/retired-systems.mdx [hunk 2]
```diff
@@ -15,6 +15,7 @@ The following systems have been retired. For the full list of available systems,
 
 | System name       | Qubit count | Retirement date (Year - month - day) |
 | ----------------- | ----------- | --------------- |
+| ibm_ithaca        | **65**      | 2024-01-25      |
```
Comment (abbycross) [inline docs/run/retired-systems.mdx:18]:
```suggestion
| ibm_ithaca        | **65**      | 2024-01-24      |
```

**Comment by abbycross** [comment]
Ithaca has been removed from IQP

**Review by beckykd — APPROVED** [review]

**Pull request merged**